code/modules/mob/living/carbon/monkey/monkey.dm

```diff
@@ -4,7 +4,7 @@
 	initial_language_holder = /datum/language_holder/monkey
 	possible_a_intents = list(INTENT_HELP, INTENT_DISARM, INTENT_HARM)
 	icon = 'icons/mob/monkey.dmi'
-	icon_state = null
+	icon_state = "monkey1"
```

Comment:
Why is this changed?

Reply:
This is an old helper for blindsense, I can probably remove it now